github/github.go

```diff
@@ -199,6 +200,13 @@ func HandlePullRequest(logger *zap.Logger, postgres db.IClaDB, payload webhook.P
 			users = append(users, " @"+v.User.Login)
 		}
 
+		// store failed users, so we can reevaluate their PR's after they sign the CLA
+		now := time.Now()
```

Comment:
If `time.Now` is single return and no error (I forget) you could just pop it in the function call below

Reply:
Noice! done: f03db01